integration/nwo/network.go

```diff
@@ -683,15 +684,17 @@ func (n *Network) Bootstrap() {
 	Eventually(sess, n.EventuallyTimeout).Should(gexec.Exit(0))
 
 	for _, c := range n.Channels {
-		sess, err := n.ConfigTxGen(commands.CreateChannelTx{
-			ChannelID:             c.Name,
-			Profile:               c.Profile,
-			BaseProfile:           c.BaseProfile,
-			ConfigPath:            n.RootDir,
-			OutputCreateChannelTx: n.CreateChannelTxPath(c.Name),
-		})
-		Expect(err).NotTo(HaveOccurred())
-		Eventually(sess, n.EventuallyTimeout).Should(gexec.Exit(0))
+		if !n.UseConfigPkg {
```

Comment:
This should be unnecessary. We can simply ignore the result of calling configtxgen.

Reply:
Done